sdks/wasm/DebuggerTestSuite/CallFunctionOnTests.cs

```diff
@@ -476,7 +476,9 @@ public async Task CheckErrorsWithSilent (bool? silent)
 
 				// callFunctionOn
 				result = await ctx.cli.SendCommand ("Runtime.callFunctionOn", cfo_args, ctx.token);
-				Assert.True ((silent ?? false) == result.IsOk);
+				Assert.True (result.IsOk);
+				Assert.False (result.IsErr);
+				Assert.True (result.ResultHasError);
```

Comment:
Shouldn't this depend on `silent`?

Reply:
In theory it should - but I couldn't find any real-life example where that would make any difference.

Reply:
I have updated that test, and the behavior for `callFunctionOn`+`silent` in https://github.com/mono/mono/pull/20044 .